gax/src/test/java/com/google/api/gax/rpc/ApiExceptionFactoryTest.java

```diff
@@ -149,6 +149,20 @@ public void unauthenticated() {
         .isInstanceOf(UnauthenticatedException.class);
   }
 
+  @Test
+  public void unimplemented() {
+    Truth.assertThat(createException(Code.UNIMPLEMENTED))
+        .isInstanceOf(UnimplementedException.class);
+    Truth.assertThat(createExceptionWithMessage(Code.UNIMPLEMENTED))
+        .isInstanceOf(UnimplementedException.class);
+  }
+
+  @Test
+  public void unknown_default() {
```

Comment:
Is there a reason this is lower underscore?

Reply:
I felt like using `unknownDefault` might imply a status code of UNKNOWN_DEFAULT, but it's really testing the default case when there is an unhandled status code.